src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -24,11 +24,17 @@ themes = {
     }
 }
 
+slider-thumb-common-contrastTheme()
+    background: none;
+    background-color: fColor;
```

Comment:
I'm assuming this is defining the stylus function. We still use the {} to keep things readable.